tests.msbuild

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It can just be tests.builds. I was probably overthinking this because it's not exactly like the other .builds files, but that doesn't really matter. I'll rename.